controllers/booksController.js

```diff
@@ -1,7 +1,7 @@
 import Book from '../model/Book.js';
 
 export const handleNewBook = async (req, res) => {
-  const { title, author } = req.body;
+  const { title, author, readOn } = req.body;
```

Comment:
added this property because we need it for rendering the "previously read" section of the FE